components/dashboard/sections/platform-subscription/DashboardPlatformSubscription.tsx

```diff
@@ -131,16 +132,16 @@ export function DashboardPlatformSubscription(props: DashboardSectionProps) {
             id="S3DC9x"
             values={{
               LegacyPricingLink: getI18nLink({
-                href: 'https://opencollective.com/pricing',
-                openInNewTabNoFollow: true,
+                as: Link,
+                href: '/pricing',
```

Comment:
Should ideally use the `NEW_PRICING` env, to automatically adapt when we release